apps/researcher/src/app/[locale]/objects/[id]/date-created-to-string.test.tsx

```diff
@@ -0,0 +1,57 @@
+import dateCreatedToString from './date-created-to-string';
+import {describe, expect, it} from '@jest/globals';
+
+describe('dateCreatedToString', () => {
+  it('returns undefined if both startDate and endDate are undefined', async () => {
+    const date = await dateCreatedToString(
+      {startDate: undefined, endDate: undefined, id: 'timeSpan'},
+      'en'
+    );
+
+    expect(date).toBeUndefined();
```

Comment:
For consistency, use the same `expect()` notation as in the other tests (`expect(dateCreatedToString(...))`)?

Personally I prefer the notation you're using in this test: first the function call, then the expect (I like the readability).